03week/news/css/style.css

```diff
@@ -1,33 +1,56 @@
 .container {
   display: flex;
 flex-wrap: wrap;
-flex-direction: column;
+flex-direction: row;
+background-color: #D0EAED;
 
 }
 
 .img{
   display: flex;
-justify-content: space-around;
+justify-content: flex-start;
   width: 50%;
   height: 50%;
 flex: 2;
+border: 2px solid black;
 }
 
 .article {
   display: flex;
 flex: 2;
-
+border: 2px solid black;
+background-color: #E0C7B3;
 }
 
 .aside {
-flex:-2;
-align-self: flex-end;
+flex: 0;
```

Comment:
I think your aside isn't showing up because the flex value should be 1 instead of 0. So article would take up 2 columns and aside would take up 1. I'm thinking that when it's set to zero the flex grid hides it.